vertx-sql-client/src/main/java/io/vertx/sqlclient/Row.java

```diff
@@ -188,6 +188,29 @@ default String getString(String column) {
     return getString(pos);
   }
 
+  /**
+   * Get a JSON value for the given {@code column}, the JSON value might be one of the following types:
+   * <ul>
+   *   <li>String</li>
+   *   <li>Number</li>
+   *   <li>JsonObject</li>
+   *   <li>JsonArray</li>
+   *   <li>Boolean</li>
+   *   <li>Null</li>
```

Comment:
`Null` is not a type, instead it should be `might be {@code null} or one of the following types:`